src/app/zap-templates/zcl/data-model/chip/thermostat-cluster.xml

```diff
@@ -501,158 +503,157 @@ limitations under the License.
       <access op="write" privilege="manage"/>
       <optionalConform/>
     </attribute>
-    <attribute side="server" code="0x0041" name="ACCapacity" define="AC_CAPACITY" type="int16u" writable="true" default="0" optional="true" max="65535">
+    <attribute side="server" code="0x0041" name="ACCapacity" define="AC_CAPACITY" type="int16u" writable="true" default="0">
       <access op="write" privilege="manage"/>
       <optionalConform/>
     </attribute>
-    <attribute side="server" code="0x0042" name="ACRefrigerantType" define="AC_REFRIGERANT_TYPE" type="ACRefrigerantTypeEnum" min="0x0" max="0x3" writable="true" default="0" optional="true">
+    <attribute side="server" code="0x0042" name="ACRefrigerantType" define="AC_REFRIGERANT_TYPE" type="ACRefrigerantTypeEnum" max="0x3" writable="true" default="0">
       <access op="write" privilege="manage"/>
       <optionalConform/>
     </attribute>
-    <attribute side="server" code="0x0043" name="ACCompressorType" define="AC_COMPRESSOR_TYPE" type="ACCompressorTypeEnum" min="0x0" max="0x3" writable="true" default="0" optional="true">
+    <attribute side="server" code="0x0043" name="ACCompressorType" define="AC_COMPRESSOR_TYPE" type="ACCompressorTypeEnum" max="0x3" writable="true" default="0">
       <access op="write" privilege="manage"/>
       <optionalConform/>
     </attribute>
-    <attribute side="server" code="0x0044" name="ACErrorCode" define="AC_ERROR_CODE" type="ACErrorCodeBitmap" writable="true" default="0" optional="true">
+    <attribute side="server" code="0x0044" name="ACErrorCode" define="AC_ERROR_CODE" type="ACErrorCodeBitmap" writable="true" default="0">
       <access op="write" privilege="manage"/>
       <optionalConform/>
     </attribute>
-    <attribute side="server" code="0x0045" name="ACLouverPosition" define="AC_LOUVER_POSITION" type="ACLouverPositionEnum" min="0x0" max="0x5" writable="true" default="0" optional="true">
+    <attribute side="server" code="0x0045" name="ACLouverPosition" define="AC_LOUVER_POSITION" type="ACLouverPositionEnum" max="0x5" writable="true" default="0">
       <access op="write" privilege="manage"/>
       <optionalConform/>
     </attribute>
-    <attribute side="server" code="0x0046" name="ACCoilTemperature" define="AC_COIL_TEMPERATURE" type="temperature" optional="true" isNullable="true">
+    <attribute side="server" code="0x0046" name="ACCoilTemperature" define="AC_COIL_TEMPERATURE" type="temperature" isNullable="true" min="-27315">
       <optionalConform/>
     </attribute>
-    <attribute side="server" code="0x0047" name="ACCapacityformat" define="AC_CAPACITY_FORMAT" type="ACCapacityFormatEnum" min="0x00" max="0x00" writable="true" default="0" optional="true">
+    <attribute side="server" code="0x0047" name="ACCapacityFormat" define="AC_CAPACITY_FORMAT" type="ACCapacityFormatEnum" max="0x00" writable="true" default="0">
```

Comment:
Is there a risk that this change to uppercase "F" in the attribute name will break compatibility?

Reply:
I am going to revert thus part.

Reply:
This will likely cause some unwanted trouble, but we should get the spec aligned on the lowercase naming then.